app/experimenter/nimbus-ui/src/lib/experiment.ts

```diff
@@ -44,15 +39,13 @@ export type StatusCheck = ReturnType<typeof getStatus>;
 
 // Common redirects used on all Edit page components
 export function editCommonRedirects({ status }: RedirectCheck) {
-  // If experiment is in review or preview, it can't be edit,
-  // but it's also not locked, so send you to the review page
-  if (status.review || status.preview) {
-    return "request-review";
+  // If experiment is released, send you to the summary page
+  if (status.released) {
```

Comment:
I see what you're doing about tracking this, but I wouldn't fold it into state I'd add it as a separate property.  And if we want a simple bool to know if something's 'released' (or 'launched') I'd hazard we should do that mapping on the backend and expose it as a bool through the API so that we can keep state > drived state mappings closer together ie in constants if/when we need to rearrange them?